app/src/main/res/layout/walkthrough_topic_summary_view.xml

```diff
@@ -20,17 +21,20 @@
     android:onClick="@{(v) -> viewModel.clickOnSummaryTile()}"
     app:roundedRectDrawableWithColor="@{viewModel.backgroundColor}">
 
-    <ImageView
+    <org.oppia.android.app.customview.LessonThumbnailImageView
       android:id="@+id/walkthrough_topic_thumbnail_image_view"
       android:layout_width="match_parent"
       android:layout_height="92dp"
       android:importantForAccessibility="no"
       android:scaleType="centerInside"
-      android:src="@{viewModel.topicSummary.topicThumbnail.thumbnailGraphic}"
+      lesson:entityId="@{viewModel.topicSummary.topicId}"
+      lesson:entityType="topic"
```

Comment:
The reason behind failure is that it is not getting passed correctly.
I tested this. You will need make following changes:
1. you can use `app` instead of `lesson` that won't create any issues.
2. In `WalkthroughTopicViewModel` inject `@TopicHtmlParserEntityType private val topicEntityType: String`
3. Send the above `topicEntityType` to this xml file via `WalkthroughTopicSummaryViewModel`

Reply:
Result:
![Screenshot_1610355156](https://user-images.githubusercontent.com/9396084/104161331-b3f41c00-5418-11eb-9a69-4d3737b7918b.png)

Reply:
Thanks, I've made the above mentioned chages
screenshot for reference
<img src="https://user-images.githubusercontent.com/54740946/104165190-0b958600-541f-11eb-8f16-6c7bdc7465f6.png" width="33%">

Reply:
Thanks.